dependencymagnet/doc.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
@atiratree I assume this is the new format for build tags in 1.17?

Reply:
yes